src/structures/TournamentManager.ts

```diff
@@ -103,6 +103,45 @@ class TournamentManager extends Base {
     return constuctedTournaments;
   }
 
+  public async getData(region?: Region, platform?: FullPlatform) {
+    const params = new URLSearchParams();
+    if (region) params.append('region', region);
+    if (platform) params.append('platform', platform);
+    params.append('teamAccountIds', this.client.user?.id!);
+    const tournaments = await this.client.http.sendEpicgamesRequest(true, 'GET',
```

Comment:
the url here seems to be wrong. it uses events-public-service-prod.ol.epicgames.com but should use events-public-service-live.ol.epicgames.com